test_pep8/indentation.py

```diff
@@ -0,0 +1,71 @@
+"""
```

Comment:
I don't think we need to test the linter. I trust that it works.

Reply:
please remove this file.

Reply:
I won't push this files to the master branch, I'm only using them to test the incremental changes in the feature branch.